nuprojs/Microsoft.ServiceFabric.AspNetCore.Abstractions/Microsoft.ServiceFabric.AspNetCore.Abstractions.nuproj

```diff
@@ -14,20 +14,133 @@
+      </File>
+      <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.AspNetCore.xml">
+        <TargetPath>lib\net471</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>lib\net472</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)\Microsoft.ServiceFabric.AspNetCore.xml">
+        <TargetPath>lib\net472</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetCore)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>lib\netcoreapp3.1</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetCore)Microsoft.ServiceFabric.AspNetCore.xml">
+        <TargetPath>lib\netcoreapp3.1</TargetPath>
+      </File>
+      <File Include="$(DropFolderNet5)Microsoft.ServiceFabric.AspNetCore.xml">
+        <TargetPath>lib\net5.0</TargetPath>
+      </File>
+      <File Include="$(DropFolderNet5)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>lib\net5.0</TargetPath>
+      </File>
+      <File Include="$(DropFolderNet6)Microsoft.ServiceFabric.AspNetCore.xml">
+        <TargetPath>lib\net6.0</TargetPath>
+      </File>
+      <File Include="$(DropFolderNet6)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>lib\net6.0</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>runtimes\win\lib\net461</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>runtimes\win\lib\net462</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>runtimes\win\lib\net47</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>runtimes\win\lib\net471</TargetPath>
+      </File>
+      <File Include="$(DropFolderNetFramework)Microsoft.ServiceFabric.AspNetCore.dll">
+        <TargetPath>runtimes\win\lib\net472</TargetPath>
       </File>
     </ItemGroup>
     
     <ItemGroup>
       <Dependency Include="Microsoft.ServiceFabric.Services">
         <Version>[$(NugetPkg_Version_Microsoft_ServiceFabric_Services)]</Version>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+      </Dependency>
+      <FrameworkReference Include="Microsoft.AspNetCore.App">
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+      </FrameworkReference>
+    </ItemGroup>
+
+    <ItemGroup>
+      <Dependency Include="Microsoft.ServiceFabric.Services">
+        <Version>[$(NugetPkg_Version_Microsoft_ServiceFabric_Services)]</Version>
+        <TargetFramework>net5.0</TargetFramework>
+      </Dependency>
+      <FrameworkReference Include="Microsoft.AspNetCore.App">
+        <TargetFramework>net5.0</TargetFramework>
+      </FrameworkReference>
+    </ItemGroup>
+
+    <ItemGroup>
```

Comment:
You could probably use <group> to reduce verbosity

Reply:
<group> tag doesn't work in nuproj files